test/e2e/testsuites/testsuites.go

```diff
@@ -998,9 +998,23 @@ func NewTestAzVolumeAttachment(azVolumeAttachment v1alpha1ClientSet.AzVolumeAtta
 			UnderlyingVolume: volumeName,
 			NodeName:         nodeName,
 			RequestedRole:    v1alpha1.PrimaryRole,
+			VolumeContext: map[string]string{
+				"controller": "azdrivernode",
+				"name":       volumeAttachmentName,
+				"namespace":  ns,
+				"namepace":   ns,
```

Comment:
Why include the misspelled `namepace` value?

Reply:
this is from the error that required it as a field. I have included it intentionally for us to fix this typo and remove the extra field. Adding these values alone did not fix the e2e issue. @mukhoakash and I are still looking into this.